ansible/roles/debops.discourse/README.txt

```diff
@@ -0,0 +1,442 @@
+Custom versions
```

Comment:
Perhaps this file could be moved to the `docs/ansible/roles/debops.discourse/` directory, named like `role-devlopment.rst` or something? Right now it will be confusing located inside of the role directory, I think.